.readthedocs.yaml

```diff
@@ -10,6 +10,14 @@ build:
   os: ubuntu-22.04
   tools:
     python: "3.10"
+  jobs:
+    pre_build:
+      jupyter nbconvert --to rst ../examples/sonata-network/sonata-network.ipynb
+      jupyter nbconvert --to rst ../examples/nmc-portal/L5TTPC2.ipynb
+      cp ../examples/sonata-network/sonata-network.rst source/
+      cp -r ../examples/sonata-network/sonata-network_files source/
+      cp ../examples/nmc-portal/L5TTPC2.rst source/nmc-portal.rst
+      cp -r ../examples/nmc-portal/L5TTPC2_files source/
```

Comment:
Ah good. I was wondering why readthedocs was not showing it :))

Since this same code needs to be called in multiple places (.tox and .readthedocs.yaml), does it make sense to turn this into a script under the `docs` directory? With that, when new ipynb notebooks added we will just need to modify a single script.

Reply:
Done in latest commit.